x-pack/plugins/endpoint/server/routes/resolver.ts

```diff
@@ -31,12 +32,21 @@ export function registerResolverRoutes(router: IRouter, endpointAppContext: Endp
     handleChildren(log)
   );
 
+  router.get(
```

Comment:
It this API endpoint used?

Reply:
Not currently no.

Reply:
Can you delete it then?

Reply:
I think it should stay, has a very clear use case for requesting additional ancestors outside of the default size.